### ISSUE TITLE
Website Ubuntu Instructions Provide Wrong Removal Commands

### ISSUE THREAD

**Issue by machineghost** [certbot/certbot#9798]
My operating system is (include version): **Ubuntu 20**

I installed Certbot with (snap, OS package manager, pip, certbot-auto, etc): **snap**

Let's stop right here, because the issue is with the documentation for this step of Certbot.  If you go to https://certbot.eff.org/instructions?ws=nginx&os=ubuntufocal you'll see these instructions:

>The exact command to do this depends on your OS, but common examples are sudo apt-get remove certbot, sudo dnf remove certbot, or sudo yum remove certbot.

Notice that none of those command use Snap ... but in the very next instruction, you're going to tell people to install *with Snap*:

>sudo snap install --classic certbot

In summary, you tell people to install with Snap, and then when they come back months later to reinstall, you give them every removal command *except* the Snap one.

It would make much more sense for the Snap removal command to be the first recommended command, since anyone who has used Certbot before will have it installed that way.

Thus, I propose modifying that section of https://certbot.eff.org/instructions?ws=nginx&os=ubuntufocal to: 

>The exact command to do this depends on your OS, but common examples are `sudo snap remove  certbot`, sudo apt-get remove certbot, sudo dnf remove certbot, or sudo yum remove certbot.

**Comment by machineghost** [comment]
Re-filed on the website repo instead: https://github.com/certbot/website/issues/798.
